main.go

```diff
@@ -172,7 +173,8 @@ func main() {
 			Spec: dsci.DSCInitializationSpec{
 				ApplicationsNamespace: dscApplicationsNamespace,
 				Monitoring: dsci.Monitoring{
-					Enabled: false,
+					ManagementState: operatorv1.Managed,
```

Comment:
just double checking that switching from "false" to operatorv1.Managed is intended? as this will change the behaviour to enable monitoring.

Reply:
Yes it is expected. We should default to monitoring enabled. I have also tagged issue in description